assets/src/blocks/amp-story-page/edit.css

```diff
@@ -153,10 +153,6 @@ div[data-type="amp/amp-story-page"] .wp-block-image {
 	display: none;
 }
 
-#amp-story-editor .components-draggable__clone {
-	position: absolute;
```

Comment:
Removing this might influence other items being dragged-dropped as well, reordering and general dragging and dropping -- are these still working as expected?

Reply:
Actually, I just tested and it looks like dragging elements is a bit off now, otherwise works as expected..

Reply:
You mean like regular elements within a page?

I thought removing that selector would cause major problems too, but I couldn't find anything. The default is `position: fixed`, which should be more precise anyway.

Reply:
That's what I see when dragging a Text block -- the draggable clone is quite far from the dragging icon.
![Screen Shot 2019-05-03 at 5 06 17 PM](https://user-images.githubusercontent.com/3294597/57146600-023cc500-6dc6-11e9-86f7-17f62378d0fb.png)

Are you seeing something else?

Reply:
Ah, saw the issue now and fixed it by making that selector more specific to the block list.